process/interface.go

```diff
@@ -127,12 +127,15 @@ type SpecialAddressHandler interface {
 	SetElrondCommunityAddress(elrond []byte)
 	ElrondCommunityAddress() []byte
 	SetConsensusData(consensusRewardAddresses []string, round uint64, epoch uint32)
-	ConsensusRewardAddresses() []string
+	ConsensusShardRewardData() *data.ConsensusRewardData
+	SetMetaConsensusData(rewardAddresses []string, round uint64, epoch uint32)
```

Comment:
can we simplify this interface ? like sending the shardID for with set and get consensus data ?

Reply:
I think it's better to have two different functions instead, as it becomes clearer.